crates/nu-command/src/database/commands/query_db.rs

```diff
@@ -54,6 +54,21 @@ stor open | query db "SELECT * FROM my_table WHERE second = :search_second" -p {
                     "second" => Value::test_int(123)
                 })])),
             },
+            Example {
+                description: "Execute a SQL query over JSONB data",
+                example: r#"stor create -t my_table -c {data: jsonb}
+stor insert -t my_table -d {data: {foo: foo, bar: 12, baz: [0 1 2]}}
+stor open | query db "SELECT data->'baz' as baz from my_table" | update baz {from json}"#,
```

Comment:
I'm not that proficient when working with sqlite but is there a way to avoid the `from json` here? Most of this PR makes working with JSON values a breeze from what I'm understanding from the value conversion functions but this is an odd one out.

Reply:
Sadly no :/ that's what I was explaining in (this comment)[https://github.com/nushell/nushell/pull/16258#issuecomment-3124748457]: here we do not select `data` directly (we return the subfield "baz") so SQLite will not give us a decltype, we will just get a Text without further information about what that text represents.

Reply:
@cptpiepmatz Just did it :)